eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs

```diff
@@ -380,6 +416,7 @@ instance Crypto c => ConwayEraPParams (ConwayEra c) where
         isValid (/= zero) ppuPoolDepositL
       , isValid (/= zero) ppuGovActionDepositL
       , isValid (/= zero) ppuDRepDepositL
+      , ppu /= emptyPParamsUpdate
```

Comment:
but I would put this in a different commit or maybe at least mention it in the commit message, in case we have to track  it down (for example for changelogs).  But it's up to you of course

Reply:
Very good point. I've put it into a separate commit